mobile/src/main/res/values/strings.xml

```diff
@@ -124,6 +124,24 @@
     <string name="info_not_set">Not set</string>
     <string name="empty_page">This page doesn\'t contain any widgets that are set to be visible</string>
 
+    <!-- Background tasks -->
+    <string name="alarm_clock">Alarm clock</string>
+    <string name="settings_alarm_clock">Send alarm clock to openHAB</string>
+    <string name="settings_alarm_clock_summary">Sends alarm clock time to a configured item</string>
+    <string name="settings_alarm_clock_howto_summary">Tap here to get hints on how to set up this feature server-side</string>
+    <string name="settings_alarm_clock_howto_url" translatable="false">https://www.openhab.org/docs/apps/android.html#alarm-clock</string>
+    <string name="settings_alarm_clock_item">Item</string>
+    <string name="waiting_for_network_to_send_alarm_clock">Waiting for network to send alarm clock</string>
+    <string name="notification_channel_background">Background</string>
+    <string name="notification_channel_background_description">Used to display pending background tasks</string>
+    <string name="notification_channel_background_error">Error in background tasks</string>
+    <string name="notification_channel_background_error_description">Used to notify about errors in background tasks</string>
+    <string name="error_sending_alarm_clock">Error sending alarm clock. Got HTTP error %d</string>
+    <string name="error_sending_alarm_clock_retry">Error sending alarm clock, but it\'ll be retried later</string>
+    <string name="error_sending_alarm_clock_item_empty">Please enter a valid Item name</string>
```

Comment:
Item -> item?

Reply:
Item should be written this way. I think we should discuss this, but rather in the forum and with @ThomDietrich. Similar issue: https://community.openhab.org/t/translation-of-openhab-terms/59818